src/TypesFinder/FindTypeFromAst.php

```diff
@@ -1,61 +0,0 @@
-<?php
-declare(strict_types=1);
-
-namespace Roave\BetterReflection\TypesFinder;
-
-use phpDocumentor\Reflection\Type;
-use phpDocumentor\Reflection\Types\ContextFactory;
-use PhpParser\Node\Name;
-use PhpParser\Node\NullableType;
-use Roave\BetterReflection\SourceLocator\Located\LocatedSource;
-
-class FindTypeFromAst
```

Comment:
[`FindTypesFromAst` is present in `1.2.0`](https://github.com/Roave/BetterReflection/blob/5d656518b027d3303dee197cec10a7fcb5b712ab/src/TypesFinder/FindTypeFromAst.php), so this is a BC break